invenio_administration/assets/semantic-ui/js/invenio_administration/search/search.js

```diff
@@ -9,38 +9,45 @@
 import { createSearchAppInit } from "@js/invenio_search_ui";
 import { SearchFacets } from "./SearchFacets";
 import { SearchResultItem } from "./SearchResultItem";
-import { SearchBarElement } from "./SearchBarElement";
 import SearchEmptyResults from "./SearchEmptyResults";
 import { SearchResultsContainer } from "./SearchResultsContainer";
 import { SearchResults } from "./SearchResults";
 import { parametrize } from "react-overridable";
 import { DropdownSort } from "@js/invenio_search_ui/components";
+import { SearchResultOptions } from "./SearchResultOptions";
+import { SearchBar } from "./SearchBarElement";
 
 const domContainer = document.getElementById("invenio-search-config");
+
+const sortColumns = (columns) =>
+  Object.entries(columns).sort((a, b) => a[1].order > b[1].order);
```

Comment:
If `order` is not defined in the entries, would this throw an exception in the browser? E.g. defining `order` to something silly.